src/middlewares/request.middleware.js

```diff
@@ -295,24 +295,24 @@ function checkChannelID (req, res, next) {
   next()
 }
 
-function addChannelFilters (req, res, next) {
-  // Assuming that if request has filter by channel we use that or else the config
-  if (req && req.body && req.body.request && req.body.request.filters && req.body.request.filters.channel) {
-    next()
-  } else {
-    filterService.getChannelSearchString(function (err, channels) {
-      if (err) {
-        LOG.error(utilsService.getLoggerData({}, 'ERROR', filename, 'addChannelFilters',
-          'failed to get channels'))
-      } else if (channels && (!_.isEmpty(channels))) {
-        req.body.request.filters.channel = channels
-        LOG.info(utilsService.getLoggerData({}, 'INFO',
-          filename, 'addChannelFilters', 'added channel filter', req.body.request.filters.channel))
-      }
-      next()
-    })
-  }
-}
+// function addChannelFilters (req, res, next) {
```

Comment:
Please document this function with *some* explanation of what this code will do.

Reply:
removed the function as it is not called now